crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
Why not use a separated file?
IMHO init files should be just for imports sentences
FYI we have disabled some lint checks in init files